src/http/request/polaris-request-headers.ts

```diff
@@ -14,16 +14,27 @@ export class PolarisRequestHeaders {
     readonly requestingSystemName: string;
 
     constructor(headers: any) {
-        this.dataVersion = headers[HeaderNames.DATA_VERSION];
-        this.isSnapshot = headers[HeaderNames.SNAPSHOT];
-        this.includeLinkedOperation = headers[HeaderNames.INCLUDE_LINKED_OPER];
-        this.snapshotPageSize = headers[HeaderNames.SNAPSHOT_PAGE_SIZE];
-        this.isPolling = headers[HeaderNames.POLLING];
-        this.requestId = headers[HeaderNames.REQUEST_ID];
-        this.upn = headers[HeaderNames.UPN];
-        this.eventKind = headers[HeaderNames.EVENT_KIND];
-        this.realityId = headers[HeaderNames.REALITY_ID];
-        this.requestingSystemId = headers[HeaderNames.REQUESTING_SYS];
-        this.requestingSystemName = headers[HeaderNames.REQUESTING_SYS_NAME];
+        const joi = require('joi');
+        this.dataVersion = joi.attempt(headers[HeaderNames.DATA_VERSION], joi.number());
```

Comment:
Why use attempt on every value? Joi.validate returning validated and formated object